jekyll/_cci2/custom-images.md

```diff
@@ -16,10 +16,28 @@ This document describes how to create and use custom Docker images with CircleCI
 
 ## Overview
 
-CircleCI 2.0 gives you access to the power and flexibility of Docker. One of the ways you can take advantage of this is to create custom Docker images for your jobs. Following are the benefits of creating a custom image:
+Docker is a first-class citizen in CircleCI
```

Comment:
first-class citizen is colloquial, could be confusing and hard to translate, the original sentence is more clear and plain language.

Reply:
First-class citizen is a [technical term](https://en.wikipedia.org/wiki/First-class_citizen) used in programming language design. However! I can use different language here because (a) it's not a strictly-defined term, and (b) I'm not even sure we're honoring the definition in how we use Docker in `config.yml`.